### PR TITLE
Feature/product delivery

### PR DESCRIPTION
**Pull request by dilshanmp7**
fixed active and inactive product delivery functionality

### CODE REVIEW

File: ProductDelivery/src/ProductDeliveryWebElement.ts [hunk 1]
```diff
@@ -272,8 +273,29 @@ export default class ProductDeliveryWebElement extends CustomHTMLBaseElement {
 		this.customDeliveryEventTableElement = this.getChildElement('delivery-event-table');
 		this.customDeliveryProfileFormElement = this.getChildElement('delivery-profile-form');
 		this.customDeliveryResultElement = this.getChildElement('delivery-result');
+		this.statusElement = this.getChildElement('#status');
 		this.customDeliveryEventTableElement.GetAllDeliveryResults();
 		this.addListeners();
+		this.UpdateProductDeliveryServiceStatus();
+	}
+
+	private UpdateProductDeliveryServiceStatus() {
```
Comment (Omar-ABDELNABY) [inline ProductDelivery/src/ProductDeliveryWebElement.ts:282]: camelCase here

File: ProductDelivery/src/ProductDeliveryWebElement.ts [hunk 2]
```diff
@@ -296,11 +318,29 @@ export default class ProductDeliveryWebElement extends CustomHTMLBaseElement {
 		this.saveButton.addEventListener("click", this.submitDeliveryForm.bind(this));
 		this.tryButton.addEventListener("click", this.tryDelivery.bind(this));
 		this.runAllFailButton.addEventListener("click", this.runAllFail.bind(this));
+		this.statusElement.addEventListener("click", this.StatusChanged.bind(this));
 		this.customDeliveryEventTableElement.addEventListener('show-result', (evt) => this.showResult(evt as CustomEvent));
 		this.customDeliveryEventTableElement.addEventListener('update-single-row', (evt) => this.updateDeliveryResultRow(evt as CustomEvent));
 		this.customDeliveryEventTableElement.addEventListener('show-spinner', (evt) => this.showResultSpinner(evt as CustomEvent));
 	}
 
+	private StatusChanged() {
```
Comment (Omar-ABDELNABY) [inline ProductDelivery/src/ProductDeliveryWebElement.ts:327]: camelCase here

**Review by Omar-ABDELNABY — CHANGES_REQUESTED** [review]

File: ProductDelivery/src/ProductDeliveryWebElement.ts (hunk 2 above)
Comment (dilshanmp7) [reply ProductDelivery/src/ProductDeliveryWebElement.ts]: done

File: ProductDelivery/src/ProductDeliveryWebElement.ts (hunk 1 above)
Comment (dilshanmp7) [reply ProductDelivery/src/ProductDeliveryWebElement.ts]: done

**Pull request merged**